src/main/webapp/scripts/forum-script.js

```diff
@@ -124,7 +138,8 @@ function createElementData(element) {
  */
 function convertTimestampToDate(timestamp) {
   const date = new Date(timestamp);
-  return date.toUTCString();
+  const indexOfYear = 16; // index before time so it is not included
+  return (date.toUTCString()).substring(0, indexOfYear);
```

Comment:
Does Date have a method to format to a pattern? If not, is there a utility that can do this?

Reply:
I changed it to get the day, month, and year and return the string: month / day / year.  I believe there is a utility to do this with specific formats also.  Should I go ahead and try that?